SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -222,7 +222,12 @@ void USpatialNetDriver::InitiateConnectionToSpatialOS(const FURL& URL)
 		bPersistSpatialConnection = URL.HasOption(*SpatialConstants::ClientsStayConnectedURLOption);
 	}
 
-	if (!bPersistSpatialConnection)
+	// This will create a Spatial Connection Manager if it is our first time connection.
+	// It will also inject a command line locater.
+	bool bCreatedNewNetDriver = GameInstance->TryStartSpatialConnection();
```

Comment:
Now that this is here, couldn't it be removed from all other places?

Reply:
Yes you might be right.